sdk/forms/src/main/java/com/klaviyo/forms/KlaviyoWebViewDelegate.kt

```diff
@@ -120,6 +120,16 @@ internal class KlaviyoWebViewDelegate : WebViewClient(), WebViewCompat.WebMessag
         super.onReceivedHttpError(view, request, errorResponse)
     }
 
+    override fun onPageFinished(view: WebView?, url: String?) {
+        super.onPageFinished(view, url)
+        // When an assetSource is specified, log whether we actually got it
+        Registry.config.assetSource?.let { expected ->
```

Comment:
and this is just a handy utility that confirms whether asset source loaded as expected in logcat